http-server/nginx-php-web-server/nginx.def

```diff
@@ -0,0 +1,55 @@
+Bootstrap: library
+From: ubuntu:16.04
+
+%help
+Usage: sudo singularity instance start \
+ -B nginx/log/error.log:/var/log/nginx/error.log \
+ -B nginx/log/access.log:/var/log/nginx/access.log \
+ -B nginx/run/nginx.pid:/run/nginx.pid \
+ -B nginx/lib/:/var/lib/nginx/ \
+ -B nginx/favicon.ico:/usr/share/nginx/html/favicon.ico \
+ -B nginx/www/html/index.php:/var/www/html/index.php \
+ -B nginx/tmp/data.txt:/tmp/data.txt \
+ -B php/php.ini:/etc/php/7.0/fpm/php.ini \
+ -B php/:/run/php \
+ -B php/log/php7.0-fpm.log:/var/log/php7.0-fpm.log \
+ nginx.sif nginx php
```

Comment:
You could also add on the `help` section what contains the container (sorry the redundancy ha!) something like: `Singularity container with Nginx version x.y.z for Ubuntu 16.04` before the `Usage:`